tests/test_camera.py

```diff
@@ -182,6 +189,64 @@ async def test_camera_device_info(hass: HomeAssistant) -> None:
     assert TEST_CAMERA_FRONT_DOOR_PERSON_ENTITY_ID in entities_from_device
 
 
+async def test_camera_enable_motion_detection(
+    hass: HomeAssistant, mqtt_mock: Any
+) -> None:
+    """Test built in motion detection."""
+
+    await setup_mock_frigate_config_entry(hass)
+
+    entity_state = hass.states.get(TEST_CAMERA_FRONT_DOOR_ENTITY_ID)
+    assert entity_state
+    assert entity_state.state == "streaming"
+    assert entity_state.attributes["supported_features"] == 2
+
+    async_fire_mqtt_message(hass, "frigate/front_door/motion/state", "ON")
+    await hass.async_block_till_done()
+
+    entity_state = hass.states.get(TEST_CAMERA_FRONT_DOOR_ENTITY_ID)
+    assert entity_state
```

Comment:
Can we verify the state contains what we want to see? (Presumably the motion detection shows up as an attribute / state?)

Reply:
I tried that in a previous commit but unfortunately it seems that `motion_detection_enabled` is not a state that is exposed by the entity (I checked the HA developer tools and don't see it there for any of the camera entities I use)

Reply:
So the service call is effectively "write only" from the perspective of the entity? OK. That's a shame.

Reply:
As far as I can tell, that is the case unfortunately